BLUESPAWN-client/src/hunt/HuntRegister.cpp

```diff
@@ -48,28 +49,31 @@ bool HuntRegister::HuntShouldRun(Hunt& hunt, vector<string> vExcludedHunts, vect
 	return true;
 }
 
+bool CallFunctionSafe(const std::function<void()>& func){
```

Comment:
This is a generic function that should probably be moved so other things can access it